exporters/pelorus/timeutil.py

```diff
@@ -62,6 +62,26 @@ def parse_guessing_timezone_DYNAMIC(timestring: str, format: str) -> datetime:
         return parsed.replace(tzinfo=timezone.utc)
 
 
+def to_epoch_from_string(
+    timestring: str, allowed_length: list[int] = [10, 13]
+) -> datetime:
+    """
+    It's really EPOCH to EPOCH conversion with datetime return object.
+
+    If timestring matches expected EPOCH format a proper datetime object is returned,
+    otherwise raises ValueError and consumer of this function probably want's to use
+    other format of timestamp.
+    """
+    epoch_date_time = timestring.split(".")[0]
+    # Try to convert to an EPOCH, but only if it's 10 or 13 digit
+    if len(epoch_date_time) not in allowed_length:
+        raise ValueError(
+            f"Tried to get epoch from not allowed string length: {timestring} / {allowed_length}"
+        )
+    else:
+        return datetime.fromtimestamp(int(epoch_date_time))
```

Comment:
I think we said we'd treat 13 digit as milliseconds, right? So we'd need to divide by 1000?

Reply:
No, everything works fine here. Function allows to pass even 1 number and then expands that to *0